assignments/arrays.js

```diff
@@ -75,15 +75,22 @@ for (let idx = 0; idx < inventory.length; idx++) {
     }
 }
 
+let car33 = inventory.filter(car => car.id === 33)[0];
```

Comment:
could use `const` here instead of `let`, in most cases default to `const` and use `let` if you have to.